packages/components/src/local/organisms/support-mapping/index.tsx

```diff
@@ -22,6 +23,14 @@ export const SupportMapping: React.FC<PropTypes> = ({
     }
   }, [mapWidth])
 
+  useEffect(() => {
```

Comment:
@lilitkarapetyan - I don't see what this is doing.  Well, it is called at few times at app loading, but that point there aren't any marker clusters. It isn't called again.

What is it doing?